src/onegov/org/forms/page.py

```diff
@@ -55,3 +59,72 @@ class PageUrlForm(ChangeAdjacencyListUrlForm):
 
     def get_model(self):
         return self.model.page
+
+
+class MovePageForm(Form):
+    """ Form to move a page including its subpages. """
+
+    parent_id = ChosenSelectField(
+        label=_("Destination"),
+        choices=[],
+        validators=[
+            InputRequired()
+        ]
+    )
+
+    def on_request(self):
+        self.request.include('common')
+        self.request.include('chosen')
+
+        pages = PageCollection(self.request.session)
+        self.iterate_page_tree(pages.roots, indent='',
+                               page_list=self.parent_id.choices)
+
+        # adding root element
+        if self.request.has_permission(pages, Private):
+            self.parent_id.choices.insert(
+                0, ('root', self.request.translate(_("- Root -")))
+            )
+
+    def iterate_page_tree(self, pages, indent='', page_list=None):
+        """
+        Iterates over the page tree and lists the elements with ident
+        to show the page hierarchy in the choices list
+        """
+        for page in pages:
+            if self.request.has_permission(page, Private):
+                item = (str(page.id), f'{indent} {unidecode(page.title)}')
```

Comment:
Not required. It is just a question how you search pages with or without accents...
Using unidecode removes all accents for selecting the parent

Reply:
Done similar to agency: https://github.com/OneGov/onegov-cloud/blob/master/src/onegov/agency/forms/agency.py#L161

Reply:
The code you mention is for sorting only so that words starting with "ö" appear near words starting with "o", but nobody expects a Topic named "Büro" being displayed as "Buro":
![Screenshot from 2023-02-27 16-06-39](https://user-images.githubusercontent.com/5045627/221600798-738e2102-88bc-48ef-b29e-03baf2c1e815.png)

Reply:
ok

Reply:
done